trending_data/src/main/webapp/map_styles.js

```diff
@@ -0,0 +1,716 @@
+      },
+    ],
+  },
+  {
+    'featureType': 'poi.park',
+    'elementType': 'labels.text.fill',
+    'stylers': [
+      {
+        'color': '#3C7680',
+      },
+    ],
+  },
+  {
+    'featureType': 'road',
+    'stylers': [
+      {
+        'visibility': 'off',
+      },
+    ],
+  },
+  {
+    'featureType': 'road',
+    'elementType': 'geometry',
+    'stylers': [
+      {
+        'color': '#304a7d',
+      },
+    ],
+  },
+  {
+    'featureType': 'road',
+    'elementType': 'labels.icon',
+    'stylers': [
+      {
+        'visibility': 'off',
+      },
+    ],
+  },
+  {
+    'featureType': 'road',
+    'elementType': 'labels.text.fill',
+    'stylers': [
+      {
+        'color': '#98a5be',
+      },
+    ],
+  },
+  {
+    'featureType': 'road',
+    'elementType': 'labels.text.stroke',
+    'stylers': [
+      {
+        'color': '#1d2c4d',
+      },
+    ],
+  },
+  {
+    'featureType': 'road.highway',
+    'elementType': 'geometry',
+    'stylers': [
+      {
+        'color': '#2c6675',
+      },
+    ],
+  },
+  {
+    'featureType': 'road.highway',
+    'elementType': 'geometry.stroke',
+    'stylers': [
+      {
+        'color': '#255763',
+      },
+    ],
+  },
+  {
+    'featureType': 'road.highway',
+    'elementType': 'labels.text.fill',
+    'stylers': [
+      {
+        'color': '#b0d5ce',
+      },
+    ],
+  },
+  {
+    'featureType': 'road.highway',
+    'elementType': 'labels.text.stroke',
+    'stylers': [
+      {
+        'color': '#023e58',
+      },
+    ],
+  },
+  {
+    'featureType': 'transit',
+    'stylers': [
+      {
+        'visibility': 'off',
+      },
+    ],
+  },
+  {
+    'featureType': 'transit',
+    'elementType': 'labels.text.fill',
+    'stylers': [
+      {
+        'color': '#98a5be',
+      },
+    ],
+  },
+  {
+    'featureType': 'transit',
+    'elementType': 'labels.text.stroke',
+    'stylers': [
+      {
+        'color': '#1d2c4d',
+      },
+    ],
+  },
+  {
+    'featureType': 'transit.line',
+    'elementType': 'geometry.fill',
+    'stylers': [
+      {
+        'color': '#283d6a',
+      },
+    ],
+  },
+  {
+    'featureType': 'transit.station',
+    'elementType': 'geometry',
+    'stylers': [
+      {
+        'color': '#3a4762',
+      },
+    ],
+  },
+  {
+    'featureType': 'water',
+    'elementType': 'geometry',
+    'stylers': [
+      {
+        'color': '#0e1626',
+      },
+    ],
+  },
+  {
+    'featureType': 'water',
+    'elementType': 'labels.text.fill',
+    'stylers': [
+      {
+        'color': '#4e6d70',
+      },
+    ],
+  },
+];
+
+const standart = [];
+
+const darkerStandart = [
+  {
+    'featureType': 'landscape',
+    'stylers': [
+      {
+        'saturation': -45,
+      },
+      {
+        'lightness': -20,
+      },
+    ],
+  },
+  {
+    'featureType': 'water',
+    'stylers': [
+      {
+        'saturation': -60,
+      },
+      {
+        'lightness': -35,
+      },
+    ],
+  },
+];
+
+const darkModeWithBoarders = [
```

Comment:
For this one, could you instead copy the `darkMode` array and add the missing elements? That would help to keep this file as short as possible. It also means that you wouldn't have to change both variables to keep them synchronised.